src/API calls/getArticles.js

```diff
@@ -1,13 +1,17 @@
 import axios from "axios";
 
-const getArticles = () => {
+const getArticles = (article_id) => {
   let baseURL =
     "https://nc-news-backend-project-5kc8.onrender.com/api/articles";
 
+    if(article_id){
+        baseURL += `/${article_id}`
```

Comment:
We shouldn't need to do this logic. If you check your back end, you have an endpoint that goes to `api/articles` and one that goes to `api/articles/:article_id` so use both of them! 😄